tests/integration/components/o-s-s/dialog-test.ts

```diff
@@ -62,6 +62,33 @@ module('Integration | Component | o-s-s/dialog', function (hooks) {
       assert.dom('.oss-dialog__footer .upf-btn--destructive').exists();
       assert.dom('.oss-dialog__header .upf-badge--error').exists();
     });
+
+    test('When the value is primary, skins are correct', async function (assert) {
+      await render(
+        hbs`<OSS::Dialog @title={{this.title}} @skin="primary" @mainAction={{this.mainAction}} @secondaryAction={{this.secondaryAction}} />`
+      );
+
+      assert.dom('.oss-dialog__footer .upf-btn--primary').exists();
+      assert.dom('.oss-dialog__header .upf-badge--primary').exists();
+    });
+  });
+
+  module('For @icon', () => {
+    test('When the value is undefined, icon is correct', async function (assert) {
```

Comment:
When the value is undefined, the fa-warning icon is used (or default icon is used)

Feel free to rework the other labels of the test file 🙈 🙏